modules/ecs/task_definitions.tf

```
@@ -1,9 +1,10 @@
 locals {
-  backend_image = "${var.backend_image}:${var.backend_image_tag}"
+  webhooks_endpoint = "https://${join("/", [var.server_domain, "webhooks"])}"
```

at this point this makes more sense imo:

```suggestion
  webhooks_endpoint = "https://${var.server_domain}/webhooks"
```

---

Yeah - happy to change it. I just had left the join because you'd done it like that and there didn't seem to be a point to changing it.

---

Done.